encord/utilities/range_utilities.py

```diff
@@ -0,0 +1,77 @@
+from typing import Union, Optional, Iterable, Set
+
+from encord.objects.frames import Ranges, Range
+
+
+class RangeManager:
```

Comment:
encord/utilities is now intended for one-off utilities for various customer use cases, rather than common code. Could you move it to encord/common?